src/sections/Gsoc/gsoc.style.js

```diff
@@ -0,0 +1,170 @@
+import styled from "styled-components";
+
+
+const GSOCSectionWrapper = styled.div`
+body{
+    max-width: 100%;
+    overflow-x: hidden;
+}
+.blue{
+    color: rgb(29, 49, 108);
+}
+element.style {
+    font-size: 0.7em;
+    padding-bottom: 40px;
+    padding-top: 10px;
+    width: 300px;
+}
+.btn {
+    background-color: #3eb5a1;
+}
+.waves-effect {
+    position: relative;
+    cursor: pointer;
+    display: inline-block;
+    overflow: hidden;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    -webkit-tap-highlight-color: transparent;
+    vertical-align: middle;
+    z-index: 1;
+    -webkit-transition: 0.3s ease-out;
+    transition: 0.3s ease-out;
+}
+.btn, .btn-large {
```

Comment:
Some some these classes won't be needed now right?

Reply:
yes right, Removed them.